ape_safe/client/__init__.py

```diff
@@ -127,7 +130,7 @@ def post_transaction(
                 b"",
             )
         )
-        post_dict: Dict = {"signature": signature.hex(), "origin": ORIGIN}
+        post_dict: Dict = {"signature": signature.hex() if signature else None, "origin": ORIGIN}
```

Comment:
```suggestion
        post_dict: dict = {"signature": signature.hex() if signature else None, "origin": ORIGIN}
```

Reply:
(many more examples)

Reply:
can just run `pyupgrade --py39-plus ape_safe/**/*.py` for those